### DISCUSSION TITLE
Why does Sigma not preview PDF covers in the thumbnails or .exe icons in the UI?

### DISCUSSION THREAD

**Discussion by Intrenxic** [aleksey-hoffman/sigma-file-manager#247] [Q&A]
I very much love this program but, I wish that it would show previews for the pdf files in the thumbnails. File Explorer does that and it's very helpful too when you're searching for pdf books. Also, why does it not show the actual icons for .exe files?

**Comment by aleksey-hoffman** [comment]
The support for previewing different types of files, including pdf, will be added in the future, as well as the exe icons. 
There's no technical problems, just needs time.
